docs/sources/tutorials/alerting-get-started/alerting-get-started.md

```diff
@@ -0,0 +1,144 @@
+# Alerting: Get started
+
+In this guide, we'll walk you through the process of setting up your first alert in just a few minutes. You'll witness your alert in action with real-time data, as well as receiving alert notifications.
+
+## Requirements
+
+Docker Compose (included in Docker for Desktop for macOS and Windows)
+Git
+
+### Set up the sample application
+
+In order to provide a more hands-on experience, you will use a real-world web application to generate real data. The app will expose metrics, which will be stored in Prometheus, a popular time series database (TSDB). And finally, in Grafana Alerting, you will build an alert rule based on the data generated. 
+
+You will need to download the files to your local machine.
+
+
+1. Clone the github.com/grafana/tutorial-environment repository.
+
+    ```
+    git clone https://github.com/grafana/tutorial-environment.git
+    ```
+
+1. Change to the directory where you cloned this repository:
+
+    ```
+    cd tutorial-environment
+    ```
+
+1. Make sure Docker is running:
+
+    ```
+    docker --version
+    ```
+
+    This command will display the installed Docker version if the installation was successful. 
+1. Start the sample application:
+
+    ```
+    docker compose up -d
+    ```
+
+    The first time you run docker compose up -d, Docker downloads all the necessary resources for the tutorial. This might take a few minutes, depending on your internet connection.
```

Comment:
```suggestion
    The first time you run docker `compose up -d`, Docker downloads all the necessary resources for the tutorial. This might take a few minutes, depending on your internet connection.
```